src/hosted-form/iframe-content/hosted-input.ts

```diff
@@ -246,4 +251,19 @@ export default class HostedInput {
             },
         });
     };
+
+    private firefoxAndSafariBugfix(targetField: HTMLInputElement): void {
```

Comment:
🍹 Would it make sense to rename this function to something more description? `firefoxAndSafariBugfix` doesn't tell me much `forceFocusForToNextInput` or something along those lines?

Reply:
Sure, how about `firefoxAndSafariForceFocusToInput`?

Reply:
You don't need the parameter because `this._input` is a property of the class. Also, please prefix the private method with an underscore.

Reply:
I think that's better, I am not sure browser specific is ideal as it could potentially impact other browsers as well right? But I think it is more descriptive 👍

Reply:
@davidchin I did it this way at first. Then I found `this` refered to `window`, hence an error raised saying `this._input` is `undefined`.

Reply:
You won't get that problem if define the `firefoxAndSafariBugfix` function like this https://github.com/bigcommerce/checkout-sdk-js/pull/1373#discussion_r829698810